enterprise-suite/es-monitor-api/prometheus.yml

```diff
@@ -251,3 +250,6 @@ scrape_configs:
         action: replace
         regex: (.*)-[0-9]+
         target_label: workload
+
+    metric_relabel_configs:
```

Comment:
Is this right? According to the docs metric_relabel_configs doesn't apply to autogenerated series like `up` - I thought we use those metrics, same with scrape time, etc.